### PR TITLE
adding new fan, fcu, and vav types, fields, subfields.

### PR DESCRIPTION
**Pull request by ccquigley**

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/FAN.yaml [hunk 1]
```diff
@@ -613,6 +613,24 @@ FAN_SS_VSC_FDPM_DRSM2X:
...
+  description: "Fan with start/stop status, variable speed control and isolation damper percentage control."
+  is_canonical: true
+  implements:
+  - FAN
+  - SS
+  - VSC
+  - IDPC
+
+FAN_SS_VSC_REFC:
+  description: "Fan with start/stop status, variable speed control and refrigerant control."
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/FAN.yaml:626]: nit: not refrigeration control, but rather refrigeration concentration control

File: ontology/yaml/resources/HVAC/entity_types/VAV.yaml [hunk 2]
```diff
@@ -21,6 +21,13 @@
 # Some exceptions may exist for the control method (e.g. VAV_ED), and
 # should be avoided if possible.
 
+VAV_SD:
+  description: "Simple VAV with no damper or temperature control."
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/VAV.yaml:25]: this description says no damper control but SD is a damper control type?

File: ontology/yaml/resources/HVAC/entity_types/VAV.yaml [hunk 3]
```diff
@@ -671,6 +690,44 @@ VAV_SD_DTM:
...
+  - HWZC
+
+VAV_DFSS_DFVSC_HWVM:
+  description: "Hydronic variable speed vav with heating water valve monitoring."
+  is_canonical: true
+  implements:
+  - VAV
+  - DFSS
+  - DFVSC
+  - HWVM
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/VAV.yaml:729]: Why is this monitoring and not control? How does the HW modulate and what does it control to?

File: ontology/yaml/resources/subfields/subfields.yaml [hunk 4]
```diff
@@ -188,6 +188,7 @@ descriptor:
   phase2: "The second (nominally B) phase of three-phase power distribution systems."
   phase3: "The third (nominally C) phase of three-phase power distribution systems."
   photocatalyst: "a material that uses absorbed light energy (usually UV) to drive desired reactions that would not otherwise proceed under ambient conditions"
+  plenum: "Ductwork attached between the zone and the air handler."
```
Comment (cstirdivant) [inline ontology/yaml/resources/subfields/subfields.yaml:191]: no need to add plenum

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 5]
```diff
@@ -3352,3 +3352,30 @@ literals:
...
+- failed_supply_air_damper_alarm:
+  - ACTIVE
+  - INACTIVE
+- plenum_static_pressure_sensor:
+    flexible_min: -249.080024
+    flexible_max: 249.080024
+- low_limit_plenum_static_pressure_setpoint:
+    flexible_min: -249.080024
+    flexible_max: 249.080024
+- low_plenum_static_pressure_alarm:
```
Comment (cstirdivant) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:3379]: this is incorrect. you need to specify that it is air_static pressure

additonally, no need to define plenum -- `low_supply_air_static_pressure_alarm` should suffice

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 6]
```diff
@@ -3352,3 +3352,30 @@ literals:
...
+- failed_zone_air_co_concentration_alarm:
+  - ACTIVE
+  - INACTIVE
+- failed_zone_air_relative_humidity_alarm:
+  - ACTIVE
+  - INACTIVE
+- failed_supply_air_damper_alarm:
+  - ACTIVE
+  - INACTIVE
+- plenum_static_pressure_sensor:
```
Comment (cstirdivant) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:3373]: same notes as other plenum fields

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 7]
```diff
@@ -3352,3 +3352,30 @@ literals:
...
+    flexible_min: -249.080024
+    flexible_max: 249.080024
+- low_limit_plenum_static_pressure_setpoint:
```
Comment (cstirdivant) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:3376]: same notes as other plenum fields

**Review by cstirdivant — CHANGES_REQUESTED** [review]
Please see individual comments. Additionally, you need to add GUIDs to all new types

File: ontology/yaml/resources/HVAC/entity_types/FAN.yaml (hunk 1 above)
Comment (cstirdivant) [reply ontology/yaml/resources/HVAC/entity_types/FAN.yaml:628]: @ccquigley please fix

Comment (ccquigley) [reply ontology/yaml/resources/HVAC/entity_types/FAN.yaml]: Done

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**